extensions/ql-vscode/src/model-editor/modeling-store.ts

```diff
@@ -69,6 +100,8 @@ export class ModelingStore extends DisposableObject {
       databaseItem,
       methods: [],
       hideModeledMethods: INITIAL_HIDE_MODELED_METHODS_VALUE,
+      modeledMethods: {},
+      modifiedMethodSignatures: new Set<string>(),
```

Comment:
TypeScript should be able to derive the type here:

```suggestion
      modifiedMethodSignatures: new Set(),
```